packages/aws-cdk-lib/aws-ec2/lib/launch-template.ts

```diff
@@ -221,6 +221,15 @@ export interface LaunchTemplateProps {
    */
   readonly launchTemplateName?: string;
 
+  /**
+   * A description for the first version of the launch template.
+   * The version descrioption must be maximum 255 characters long, including hyphens (-),
+   * underscores (_), spaces, and tabs.
```

Comment:
`including hyphens (-),`
`   * underscores (_), spaces, and tabs.`

Is there documentation available to confirm this? (I could not find that in the CFn documentation.)

https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-ec2-launchtemplate.html#cfn-ec2-launchtemplate-versiondescription